wdl-lint/src/rules/import_sort.rs

```diff
@@ -79,27 +87,69 @@ impl Visitor for ImportSortRule {
             return;
         }
 
-        // Reset the visitor upon document entry
         *self = Default::default();
 
-        let imports = doc
+        // Collect all import statements
+        let imports: Vec<_> = doc
             .syntax()
             .children_with_tokens()
-            .filter(|c| c.kind() == SyntaxKind::ImportStatementNode)
-            .map(|c| c.into_node().unwrap());
-
-        let mut prev_import: Option<SyntaxNode> = None;
-        for import in imports {
-            if let Some(prev) = prev_import {
-                if import.text().to_string() < prev.text().to_string() {
-                    // Since this rule can only be excepted in a document-wide fashion,
-                    // if the rule is running we can directly add the diagnostic
-                    // without checking for the exceptable nodes
-                    state.add(import_not_sorted(import.text_range().to_span()));
-                    return; // Only report one sorting diagnostic at a time.
-                }
-            }
-            prev_import = Some(import);
+            .filter_map(|c| c.into_node())
+            .filter(|n| n.kind() == SyntaxKind::ImportStatementNode)
```

Comment:
nit: can you swap the ordering of these two? The `SyntaxKind` filter should occur first.